src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -202,6 +203,27 @@ public void worldOfTanksDrive(double forward, double backward, double rotate) {
     } else {
       drive.arcadeDrive(0, rotate);
     }
+    if (isLowGearOptimal()) {
+      shiftLow();
```

Comment:
verify that calling this over and over again when it is already in low gear is fine.